cibuildwheel/linux.py

```diff
@@ -129,72 +123,31 @@ def build(project_dir, package_name, output_dir, test_command, test_requires, be
             gid=os.getgid(),
         )
 
-        container_name = 'cibuildwheel-{}'.format(uuid.uuid4())
+        def run_docker(command, stdin_str=None):
+            print('docker command: docker {}'.format(' '.join(map(shlex_quote, command))))
+            if stdin_str is None:
+                return subprocess.check_call(['docker'] + command)
+            else:
+                process = subprocess.Popen(['docker'] + command,
+                                           stdin=subprocess.PIPE, universal_newlines=True)
```

Comment:
Just a style thing - I'm not 100% on moving the 'docker' part of the command into this function, because then you have to imagine the 'docker' word when reading the invocations. But happy to leave this if you prefer it as it is

Reply:
I don't know (and kind of expected a reaction on that). The main thing was that it was a bit weird to call the function `run_docker` when docker needed to be added manually. And that it would slightly shorten the lines actually using it (since you're already reading `run_docker`).

Happy to remove this part again, if you want? Or go ahead and push it yourself to this branch, if that's the only thing remaining after the other fixes.